sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ExtRefServiceTest.java

```diff
@@ -24,17 +28,24 @@ void updateAllExtRefIedNames_should_succeed() throws Exception {
         SCL scd = SclTestMarshaller.getSCLFromFile("/scd-extref-iedname/scd_set_extref_iedname_success.xml");
         // When
         SclReport sclReport = ExtRefService.updateAllExtRefIedNames(scd);
-        // Then : The ExtRef iedName should be updated from "IED_NAME1" to "matched FlowId1"
+        // Then :
+        // - The ExtRef iedName should be updated from "System_Version_IED_NAME2" to "IED_NAME2"
```

Comment:
why this comment, is it not what the method is supposed to do?

Reply:
That comes from a comment of @gleizesDor on previous issue : https://github.com/com-pas/compas-sct/pull/173#discussion_r997063362
Maybe we can discuss that together.